src/AppBundle/Form/PostType.php

```diff
@@ -39,7 +39,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         //     $builder->add('title', null, array('required' => false, ...));
 
         $builder
-            ->add('title', null, array('label' => 'label.title'))
+            ->add('title', null, array(
+                'attr' => array('autofocus' => true),
```

Comment:
@rasanga, I agree with you. Keeping attributes on the template is more flexible approach than this. But this PR just follows the existing code style. I think you could refactor the form templates and create a PR.